README.md

```diff
@@ -60,9 +60,11 @@ Options:
 
 Option 1 (recommended): Install the flatpak from [Flathub][flathub]: `flatpak --user install com.adilhanney.saber`.
 
-Option 2: Download `Saber.flatpak` from the latest [Release][releases] and install with `flatpak --user install Saber.flatpak`
+Option 2: Install the snap from <a href="https://snapcraft.io/saber">Snap-Store</a>
```

Comment:
Please can you make the snap Option 4 since I won't be maintaining it?
Also can you add that it's unofficially maintained

Reply:
I will maintain it continiously. It'll anyways need very less manual interuption.

Reply:
I'd still prefer it as Option 4 here, plus mentioning that it's unofficially maintained